react-components/src/utilities/getInstancesFromClick.ts

```diff
@@ -0,0 +1,146 @@
+/*!
+ * Copyright 2024 Cognite AS
+ */
+
+import {
+  type CadIntersection,
+  type ClassicDataSourceType,
+  type DataSourceType,
+  type DMDataSourceType,
+  type PointCloudIntersection
+} from '@cognite/reveal';
+import { type InstanceReference } from '../data-providers';
+import { type CdfCaches } from '../architecture/base/renderTarget/CdfCaches';
+import { fetchAncestorNodesForTreeIndex } from '../components/CacheProvider/requests';
+import { EMPTY_ARRAY } from './constants';
+import { fetchAnnotationsForModel } from '../hooks/pointClouds/fetchAnnotationsForModel';
+import { isDMIdentifier } from '../components';
+import { is360ImageAnnotation } from './is360ImageAnnotation';
+import { type RevealRenderTarget } from '../architecture';
+
+export async function getInstancesFromClick(
```

Comment:
Move this to the same folder as PointsOfInterestTool

Reply:
I want this to be a general function that can be used multiple places. I think it will be easier for others to find if it's here